.github/workflows/main.yml

```diff
@@ -93,8 +96,13 @@ jobs:
           name: code-coverage-report
           path: htmlcov
 
+  build:
+    runs-on: ubuntu-latest
+    needs: ['test']
+    if: github.event_name == 'push' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/v'))
```

Comment:
Do you know which docker image tag will be used here when the image is pushed?

Reply:
Good question. I assumed that `tag_with_ref` would take the updated ref and use it. But looking at the docs there is no info about it and a quick google search seems it's deprecated [here](https://github.com/docker/build-push-action/issues/116). I'll update the action to use the newer data.

Reply:
Fixed updating to the lastest docker actions.